examples/multiple_windows/lib/app/regular_window_edit_dialog.dart

```diff
@@ -0,0 +1,201 @@
+  final VoidCallback onClose;
+
+  @override
+  State<StatefulWidget> createState() => _RegularWindowEditDialogState();
+}
+
+class _RegularWindowEditDialogState extends State<_RegularWindowEditDialog> {
+  late Size initialSize;
+  late String initialTitle;
+  late bool initialFullscreen;
+  late bool initialMaximized;
+  late bool initialMinimized;
+
+  late final TextEditingController widthController;
+  late final TextEditingController heightController;
+  late final TextEditingController titleController;
+
+  bool? nextIsFullscreen;
+  bool? nextIsMaximized;
+  bool? nextIsMinized;
+
+  void _init() {
+    widget.controller.addListener(_onNotification);
+    initialSize = widget.controller.contentSize;
+    initialTitle = widget.controller.title;
+    initialFullscreen = widget.controller.isFullscreen;
+    initialMaximized = widget.controller.isMaximized;
+    initialMinimized = widget.controller.isMinimized;
+
+    widthController = TextEditingController(text: initialSize.width.toString());
+    heightController = TextEditingController(
+      text: initialSize.height.toString(),
+    );
+    titleController = TextEditingController(text: initialTitle);
+    nextIsFullscreen = null;
+    nextIsMaximized = null;
+    nextIsMinized = null;
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _init();
+  }
+
+  @override
+  void didUpdateWidget(covariant _RegularWindowEditDialog oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (oldWidget.controller != widget.controller) {
+      oldWidget.controller.removeListener(_onNotification);
+      _init();
+    }
+  }
+
+  void _onNotification() {
+    // We listen on the state of the controller. If a value that the user
+    // can edit changes from what it was initially set to, we invalidate
+    // their current change and store the new "initial" value.
+    if (widget.controller.contentSize != initialSize) {
+      initialSize = widget.controller.contentSize;
+      widthController.text = widget.controller.contentSize.width.toString();
+      heightController.text = widget.controller.contentSize.height.toString();
+    }
+    if (widget.controller.isFullscreen != initialFullscreen) {
+      setState(() {
+        initialFullscreen = widget.controller.isFullscreen;
+        nextIsFullscreen = null;
+      });
+    }
+    if (widget.controller.isMaximized != initialMaximized) {
+      setState(() {
+        initialMaximized = widget.controller.isMaximized;
+        nextIsMaximized = null;
+      });
+    }
+    if (widget.controller.isMinimized != initialMinimized) {
+      setState(() {
+        initialMinimized = widget.controller.isMinimized;
+        nextIsMinized = null;
+      });
+    }
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    widget.controller.removeListener(_onNotification);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: Text("Edit Window Properties"),
```

Comment:
Please prefer single quotes for Dart strings instead of double quotes. We only use double quotes if the string itself contains a single quote.

```suggestion
      title: Text("Edit Window Properties"),
```

Reply:
Same comment for other occurrences in this PR